meson.build

```diff
@@ -0,0 +1,27 @@
+project('tinyendian', 'd')
+
+project_version      = '0.1.2'
+project_soversion    = '0'
+
+src_dir = include_directories('source/')
+pkgc = import('pkgconfig')
+
+tinyendian_src = [
+    'source/tinyendian.d'
+]
+install_headers(tinyendian_src, subdir: 'd/')
+
+tinyendian_lib = static_library('tinyendian',
+        [tinyendian_src],
+        include_directories: [src_dir],
+        install: true,
+        version: project_version,
+        soversion: project_soversion,
+        pic: true
```

Comment:
Is specifying `-fPIC` manually really needed? As far as I know, both DMD and LDC have had `-fPIC`  enabled by default on Linux 64 for several versions - see:
https://github.com/dlang/dmd/blob/v2.074.0/ini/linux/bin64/dmd.conf
https://github.com/ldc-developers/ldc/pull/1664/files#diff-392adbd5df3bd90c4820551ceb054e6bR535